test/testdata/resolver/suggest_unsafe_override.rb

```diff
@@ -34,7 +34,11 @@ class Child3 < Parent
   extend T::Sig
   extend T::Helpers
 
+  # The error-with-dupes below is because `--suggest-unsafe` runs `sig_finder`,
+  # which calls type_syntax parsing, which unconditionally reports errors.
+  # We probably want some way to swallow errors when sig_finder runs type_syntax parsing.
   sig {override(allow_incompatible: false).params(x: T::Boolean).void}
-#                                                 ^ error: Parameter `x` of type `T::Boolean` not compatible with type of overridden method `Parent#foo`
+  #                                 ^^^^^ error-with-dupes: `override(allow_incompatible: ...)` expects either `true` or `:visibility`
```

Comment:
This is the line I'm re-requesting review for. We duplicate this error in `--suggest-unsafe`, because it shows up when we run sig_finder.